xml/vk.xml

```diff
@@ -2568,6 +2568,55 @@ typedef void <name>CAMetalLayer</name>;
             <member><type>float</type>   <name>x</name></member>
             <member><type>float</type>   <name>y</name></member>
         </type>
+        <type category="struct" name="VkSwapchainTimingPropertiesEXT">
+            <member values="VK_STRUCTURE_TYPE_SWAPCHAIN_TIMING_PROPERTIES_EXT"><type>VkStructureType</type> <name>sType</name></member>
+            <member>const <type>void</type>*                      <name>pNext</name></member>
+            <member><type>uint64_t</type>                         <name>refreshDuration</name><comment>Number of nanoseconds from the start of one refresh cycle to the next</comment></member>
+            <member><type>VkBool32</type>                         <name>variableRefresh</name></member>
+        </type>
+        <type category="struct" name="VkSwapchainTimeDomainPropertiesEXT">
+            <member values="VK_STRUCTURE_TYPE_SWAPCHAIN_TIME_DOMAIN_PROPERTIES_EXT"><type>VkStructureType</type> <name>sType</name></member>
+            <member><type>void</type>*                            <name>pNext</name></member>
+            <member><type>VkTimeDomainEXT</type>                  <name>timeDomain</name><comment>Available time domain to use with the swapchain</comment></member>
+        </type>
+        <type category="struct" name="VkSwapchainTimingInfoEXT">
+            <member values="VK_STRUCTURE_TYPE_SWAPCHAIN_TIMING_INFO_EXT"><type>VkStructureType</type> <name>sType</name></member>
+            <member>const <type>void</type>*                      <name>pNext</name></member>
+            <member><type>VkTimeDomainEXT</type>                  <name>timeDomain</name><comment>Available time domain to use with the swapchain</comment></member>
+        </type>
+        <type category="struct" name="VkPastPresentationTimingEXT">
```

Comment:
Why is the presentMargin from VK_GOOGLE_display_timing missing here?

Reply:
A +1 on this, or is it expected that the application use `vkCmdWriteTimestamp` to calculate margins on their own? Should text be added and maybe a special `VkPipelineStageFlagBits` flag to signal that the work is specifically done for presentation?